configs/openmrs_config/roles/cambodia_roles.csv

```diff
@@ -1,2 +0,0 @@
-Uuid,Role name,Description,Inherited roles,Privileges,_order:1100
```

Comment:
I'd leave the file name untouched to avoid potential collusion from parent configs.